dev-lang/cfortran/cfortran-20110621.ebuild

```diff
@@ -38,6 +38,11 @@ src_prepare() {
 	fi
 }
 
+src_configure() {
+	use sparc && append-fflags $(test-flags-FC -fno-store-merging -fno-tree-slp-vectorize)
```

Comment:
Please add #bug references to make it easier to check in future. (Could you do this one in a follow up PR? Feel free to throw it in as a separate commit with other changes to other packages in another PR.)

Reply:
Oops, I usually do this, just forgot this time.  I will stable arm64 and do it there.  (Last one is hppa which I don't have...yet)